int8 angle_distortion_correction
---
bool result
